config/configmodels/configmodels.go

```diff
@@ -160,10 +150,6 @@ type Service struct {
 type ReceiverSettings struct {
 	TypeVal Type   `mapstructure:"-"`
 	NameVal string `mapstructure:"-"`
-	// Configures if the receiver is disabled and doesn't receive any data.
-	// The default value is false(meaning the receiver is enabled by default), and it is expected that receivers
-	// continue to use the default value of false.
-	Disabled bool `mapstructure:"disabled"`
 	// Configures the endpoint in the format 'address:port' for the receiver.
 	// The default value is set by the receiver populating the struct.
 	Endpoint string `mapstructure:"endpoint"`
```

Comment:
while you are at it, might as well fix the doc to start with 'Endpoint'.

Reply:
Done.